main.py

```diff
@@ -72,7 +92,11 @@ def execute_script(script_path):
 
     node_paths = folder_paths.get_folder_paths("custom_nodes")
     for custom_node_path in node_paths:
-        possible_modules = os.listdir(custom_node_path)
+        if not os.path.exists(custom_node_path):
+            logging.warning(f"Directory {custom_node_path} for custom nodes does not exist, skipping")
+            continue
```

Comment:
This is the primary fix I'm interested in, if the directory does not need to exist but is checked by default, it seems better to skip instead of crash?

I am not sure if that aligns with your expectations, so if it makes sense to log a warning I've added one here, otherwise the warning could be removed?